pontoon/base/static/js/translate.js

```diff
@@ -109,9 +110,14 @@ var Pontoon = (function (my) {
         '</p>' +
         '<span class="arrow fa fa-chevron-right fa-lg"></span>' +
         '</li>', self.app.win);
+      console.log('renderEntity: '  + entity.pk);
+      console.log('searchQuery: ' + searchQuery);
```

Comment:
'k

On Wed, Sep 7, 2016 at 2:46 AM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/base/static/js/translate.js
> <https://github.com/mozilla/pontoon/pull/462#discussion_r77744656>:
>
> > @@ -109,9 +110,14 @@ var Pontoon = (function (my) {
> >          '</p>' +
> >          '<span class="arrow fa fa-chevron-right fa-lg"></span>' +
> >          '</li>', self.app.win);
> > +      console.log('renderEntity: '  + entity.pk);
> > +      console.log('searchQuery: ' + searchQuery);
>
> You can remove this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/pontoon/pull/462/files/fff5e5d08e23dd52ec88f5cf29d6aa90bb4a8178#r77744656>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIYMTMIQtTLdilboWju8JAQOprQc4RR3ks5qnglWgaJpZM4J2V6T>
> .
>



-- 
Kind regards

​Jarek "jotes" Śmiejczak,
​P​
ython programmer​ by day, mozillian by heart..
Homepage: http://joyfulhack.in, Github: http://github.com/jotes
Mobile: +48693027040, Mozillian: http://mozillians.org/u/jotes